Update surface doc for apriltags (#265)
pupil-labs/pupil-docs: user-docs/pupil-capture.md

### Surface Tracking

The `Surface Tracker` plugin allows you to define planar surfaces within your environment to track areas of interest (AOI). Surfaces are defined with [Apriltag Markers](https://april.eecs.umich.edu/software/apriltag.html).

<aside class="notice">

>   <strong>Note</strong> - The legacy surface system used simple square markers, which are way less robust to detect.

Reviewer: ```suggestion
  <strong>Note</strong> - The legacy surface tracker used simple square markers, which are less robust to detect.
```

  For all new projects we strongly recommend using Apriltags.
  You can find more information on the legacy markers below.
</aside>

#### Markers

There are many different apriltag types, currently we only support apritags of type **tag36h11**, which is the recommended set. There are 587 distinct markers in the set, so it should be sufficient for most use cases.

> You can find images of all markers in the [original Apriltag GitHub repository](https://github.com/AprilRobotics/apriltag-imgs/tree/master/tag36h11), but you will have to upscale them, as they are scaled to minimal size. We have two pages of markers prepared for easy use on the right, that you can just download to get started.

Reviewer: ```suggestion
You can find images of all markers in the [original Apriltag GitHub repository](https://github.com/AprilRobotics/apriltag-imgs/tree/master/tag36h11). The tag images are very small in the apriltags repo. Therefore you will need to scale them up before use. We have two pages of markers prepared convenience on the right, that you can download to get started.
```

Markers can be printed on paper, stickers, or displayed on a screen.

> <div class="figure-container">
	<div class="Grid Grid--1of2 img-row">
		<div class="Grid-cell" style="padding-right:5px;">
			{{< lqip-img figure-class="img-m" src="/images/pupil-capture/calibration-markers/apriltags_tag36h11_0-23.jpg" alt="Apriltags tag36h11 0-23" >}}
			<p>Apriltags tag36h11 0-23</p>
		</div>
		<div class="Grid-cell">
			{{< lqip-img figure-class="img-m" src="/images/pupil-capture/calibration-markers/apriltags_tag36h11_24-47.jpg" alt="Apriltags tag36h11 24-47" >}}
			<p>Apriltags tag36h11 24-47</p>
		</div>
	</div>
</div>

<aside class="notice">
  <strong>Note</strong> - When displaying or printing markers, ensure that a small white border remains around the marker! 

>   The border should be around as wide as a cell of the grid size of the marker. 

Reviewer: ```suggestion
  The border should be around as wide as one grid cell of the marker. 
```

*   Streaming Surfaces with Pupil Capture - Detected surfaces as well as gaze positions relative to the surface are broadcast under the `surface` topic. Check out [this video](http://youtu.be/qHmfMxGST7A) for a demonstration.
*   Surface Metrics with Pupil Player - if you have defined surfaces, you can generate surface visibility reports or gaze count per surface. See our [blog post](http://pupil-labs.com/blog/2014/07/0392-player-release.html) for more information.

#### Legacy Markers

> The legacy surface system used simple square markers, which are way less robust to detect.

Reviewer: ```suggestion
The legacy surface system used simple square markers, which are less robust to detect.
```

For all new projects we strongly recommend using Apriltags!

<aside class="warning">

>   <strong>Note</strong> - The legacy marker system had a problem in pupil player, where marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. When working with legacy markers, you are strongly recommended to always only enable the offline surface detection in pupil player when the seek position is at frame 0 and to not seek in the video as long as the surface detection is still running, in order to avoid any data inconsistencies!

Reviewer 2: ```suggestion
  <strong>Note</strong> - The legacy marker system has a problem in Pupil Player, where marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. When working with legacy markers, you are strongly recommended to always only enable the offline surface detection in Pupil Player when the seek position is at frame 0 and to not seek in the video as long as the surface detection is still running, in order to avoid any data inconsistencies!
```

>   <strong>Note</strong> - The legacy marker system has a problem in Pupil Player, where marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. When working with legacy markers, you are strongly recommended to always only enable the offline surface detection in Pupil Player when the seek position is at frame 0 and to not seek in the video as long as the surface detection is still running, in order to avoid any data inconsistencies!

Reviewer: This section needs to be re-worked IMO. 

```suggestion
  <strong>Note</strong> - When working with legacy markers in Pupil Player, you are strongly recommended to only enable the offline surface detection in Pupil Player when the seek position is at frame 0 and not to seek in the video as long as the surface detection is still running. Explanation: There is a bug in the legacy marker tracking system in Pupil Player. Marker detection could be non-unique if the marker detection process was not started from the beginning of the recording or was discontinued while detecting by seeking in the recording. 
```

not the best edit, but maybe could be improved by @papr or @pfaion

Author: I like it. It should not bee too important anyways, as users are strongly recommended to switch to apriltags for a lot of other reasons anyways. We were not even sure about whether this is a good place to include this warning.
